master/internal/resourcemanagers/resource_managers_test.go

```diff
@@ -35,3 +36,30 @@ func TestResourceManagerForwardMessage(t *testing.T) {
 	assert.DeepEqual(t, taskSummary, make(map[sproto.TaskID]TaskSummary))
 	assert.NilError(t, rpActor.StopAndAwaitTermination())
 }
+
+func TestResourceManagerValidateRPResourcesUnknown(t *testing.T) {
+	// We can reliably run this check only for AWS, GCP, or Kube resource pools,
+	// but initializing either of these in the test is not viable. So let's at least
+	// check if we properly return "unknown" for on-prem-like setups.
+	system := actor.NewSystem(t.Name())
+	conf := &ResourceConfig{
+		ResourceManager: &ResourceManagerConfig{
+			AgentRM: &AgentResourceManagerConfig{
+				Scheduler: &SchedulerConfig{
+					FairShare:     &FairShareSchedulerConfig{},
+					FittingPolicy: best,
+				},
+			},
+		},
+		ResourcePools: []ResourcePoolConfig{
+			{
+				PoolName:                 defaultResourcePoolName,
+				MaxAuxContainersPerAgent: 100,
+			},
+		},
+	}
+
+	Setup(system, echo.New(), conf, nil, nil)
+	value := sproto.ValidateRPResources(system, defaultResourcePoolName, 1)
+	assert.Assert(t, is.Nil(value))
```

Comment:
Why not just `assert.Assert(t, value == nil)` or assert? (ask mainly because the rename import is not standard go).

Reply:
i've looked at using testify, fine with it tbh